playbooks/roles/wireguard/templates/instructions.md.j2

```diff
@@ -35,3 +41,34 @@ WireGuard is currently only available for Linux, but [cross-platform](https://ww
 
 #### A note on DNS configuration ####
 Each client configuration profile includes a `DNS` command that uses resolvconf to direct DNS traffic to the dnsmasq server that is available via the WireGuard encrypted interface at `{{ dnsmasq_wireguard_ip }}`. Although resolvconf is a common utility, you may need to use `PostUp`/`PostDown` with a different command for your distribution or particular network configuration.
+
+
+<a name="macos"></a>
+### macOS ###
+**WARNING**: Wireguard macOS client is in early stages of development and still considered experimental. It may be unstable or buggy.
+
+
+1. Install [Homebrew](https://brew.sh/), if you haven't already.
+1. Install Wireguard Tools using Homebrew:
+
+   `brew install wireguard-tools`
+1. Download one of the client configuration files. **Only one computer can use a profile at a time**. For this example we'll assume you downloaded {{ vpn_client_names.results[0].stdout }}:
+{% for client in vpn_client_names.results %}
+   * [{{ client.stdout }}](/wireguard/{{ client.stdout }}.conf)
+{% endfor %}
+1. Move the client configuration file into the correct directory (this command assumes that you downloaded the configuration file in your Downloads folder. If that's not the case, modify it accordingly):
+
+   `sudo sh -c 'umask 077; mkdir -p /etc/wireguard; cat > /etc/wireguard/{{ vpn_client_names.results[0].stdout }}.conf' < ~/Downloads/{{ vpn_client_names.results[0].stdout }}.conf`
```

Comment:
A style point: Please use `&&` instead of `;` to chain commands.

Consider putting the filename inside the sudo:

```
sudo sh -c 'umask 077 && mkdir -p /etc/wireguard/ && cat poem-walk.conf >/etc/wireguard/poem-walk.conf'
```

This could be broken up into two commands and it would be clearer:
```
sudo mkdir -m 0700 -p /etc/wireguard/
sudo install -m 0600 poem-walk.conf /etc/wireguard/
```

Some people don't like `install`, though.